client/lib/analytics/index.js

```diff
@@ -574,6 +586,11 @@ const analytics = {
 			label,
 			value
 		) {
+			if ( 'undefined' !== typeof value && /^[0-9.]+$/.test( String( value ) ) ) {
```

Comment:
I wonder if this test for "numericness" would be less brittle than a regular expression, as numbers can come in many different formats: 

`! isNaN( Number( String( value ) ) )`

Reply:
Agree. That seems better to me too. Thank you.

Reply:
@lsinger Updated to `! isNaN( Number( String( value ) ) )`. Thanks again!